systemd/ceph-osd@.service.in

```diff
@@ -9,7 +9,7 @@ Environment=CLUSTER=ceph
 EnvironmentFile=-@SYSTEMD_ENV_FILE@
 ExecReload=/bin/kill -HUP $MAINPID
 ExecStart=/usr/bin/ceph-osd -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
-ExecStartPre=/usr/lib/ceph/ceph-osd-prestart.sh --cluster ${CLUSTER} --id %i
+ExecStartPre=/usr/libexec/ceph/ceph-osd-prestart.sh --cluster ${CLUSTER} --id %i
```

Comment:
This should use `@CMAKE_INSTALL_FULL_LIBEXECDIR@` instead of a hardcoded path and get substitutions from CMake accordingly...

Cf. https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html

Reply:
Like so?

Reply:
Exactly like you did. 👍